lib/About.dart

```diff
@@ -121,3 +121,5 @@ class About extends StatelessWidget {
     );
   }
 }
+
+//TODO: Create a Dart File containing several Text widgets so that we can make changes to paras or texts from  a single place.
```

Comment:
Sir , I had made that comment during the initial stage . But I thought if someone else would be working on this . It was not in my assigned issue but I added it if it would be helpful in maintaining this project.

Reply:
You can always create a separate issue for that but please don't add any commented code. :)

Reply:
Okay sir got it . I was not aware about this but okay now I will not do that again